canvas_modules/common-canvas/src/palette/palette-flyout-content-search.jsx

```diff
@@ -50,7 +50,7 @@ class PaletteFlyoutContentSearch extends React.Component {
 
 		return (
 			// palette-flyout-search id added for hopscotch tours
-			<div className="palette-flyout-search-container" id="palette-flyout-search">
+			<div className="palette-flyout-search-container" id="palette-flyout-search-container">
```

Comment:
@srikant-ch5 `palette-flyout-search` id is used in hopscotch tour in SPSS. Changing this id will break the tour.

Reply:
Thanks @nmgokhale. I have reverted the change and closing this PR as few other Accessibility Issue changes were merged by Prince's PR.